testsuite/MDAnalysisTests/util.py

```diff
@@ -148,3 +149,50 @@ def in_dir(dirname):
     os.chdir(dirname)
     yield dirname
     os.chdir(old_path)
+
+
+def assert_nowarns(warning_class, *args, **kwargs):
```

Comment:
Have we got tests for the functions that we define for our test suite?

It has nothing to do with `test_util.py`, which tests MDAnalysis.lib.util.